boards/native/board_init.c

```diff
@@ -79,5 +80,6 @@ VFS_AUTO_MOUNT(native, { .hostpath = FS_NATIVE_DIR }, VFS_DEFAULT_NVM(0), 0);
  */
 void board_init(void)
 {
+    setenv("TZ", "UTC", 0);
```

Comment:
Let's guard this behind a pseudo-module or config option that gets selected by `Makefile.tests_common`

Reply:
how did you set the rtc ? (we should probably start it at 0 where most of our boards start) :)

Reply:
I did not set it at all, it's synced with the system time by default.

Reply:
I like sane defaults and the time functions are insane and even more so if timezone things are happening lets ditch then in a more sane direction by default.

if you like your localtime to much `TZ=/etc/localtime <elf_file>` or `TZ=/etc/localtime make term or test`
is an option
using mktime on native is a gamble otherwise maybe we should run a Linter that warns and errors about that.

Reply:
IMHO `native` showing the same time as the host system *is* a sane default.
Why not make it a config option so we can have predictive behavior for tests that need it without forcing everyone to accept UTC as the one true time?

Reply:
- cause in networking our timestamps are usually UNIX time - which is kind of UTC.
- since RIOT is an IoT OS  we should probably be as compatible as we can.

Reply:
https://www.sourceware.org/git/?p=newlib-cygwin.git;a=blob;f=newlib/libc/time/tzvars.c;h=d1f17649b7112ea520032f2bfc109191febb5a2c;hb=HEAD#l5